sympy/plotting/plot.py

```diff
@@ -293,32 +293,30 @@ class PlotGrid(object):
     """This class helps to plot subplots from already created sympy plots
     in a single figure
     """
-    def __init__(self, nrows, ncolumns, *args, show=True):
+    def __init__(self, nrows, ncolumns, *args, **kwargs):
```

Comment:
Yes, this is an issue with 2.7... Could one possibly consider having the plots in a list? Not sure what is the better option.